internal/lookup/lookup.go

```diff
@@ -132,13 +137,19 @@ func storeUserdata(ctx context.Context, db *sqlx.DB, logger *zap.Logger, lookupR
 		Userdata: null.NewBytes(lookupResp.Userdata, true),
 	}
 
-	err := upserter.UpsertUserdata(ctx, db, logger, lookupResp.ID, lookupResp.IPAddresses, newInstanceUserdata)
-	if err != nil {
-		middleware.MetricUserdataStoreErrors.Inc()
-		return nil, err
+	if viper.GetBool("crdb.enabled") {
```

Comment:
nit: The string "crdb.enabled" is used multiple places. It would be more maintainable if this were a constant.